test/Microsoft.Azure.WebJobs.Host.UnitTests/Common/BindingFactoryHelperTests.cs

```diff
@@ -1,38 +0,0 @@
-﻿// Copyright (c) .NET Foundation. All rights reserved.
-// Licensed under the MIT License. See License.txt in the project root for license information.
-
-using System;
-using Microsoft.Azure.WebJobs.Host.Config;
-using Microsoft.Azure.WebJobs.Host.TestCommon;
-using Xunit;
-using System.Threading.Tasks;
-using System.Reflection;
-using Microsoft.Azure.WebJobs.Host.Bindings;
-
-namespace Microsoft.Azure.WebJobs.Host.UnitTests.Common
-{
-    public class BindingFactoryHelperTests
```

Comment:
It's in TypedCollectorTests.cs

---
In reply to: [99682130](https://github.com/Azure/azure-webjobs-sdk/pull/1006#discussion_r99682130) [](ancestors = 99682130)